flink/src/test/java/org/apache/sedona/flink/FunctionTest.java

```diff
@@ -699,4 +699,14 @@ public void testNumPoints() {
         assertEquals(expected, actual);
     }
 
+    @Test
```

Comment:
In Sedona Flink, the ST_Force3D has 2 overloaded `eval` functions. Please test both funcs in the test.

Reply:
Added